### ISSUE TITLE
Is it possible to write iOS client, that uses model served with TensorFlow Serving

### ISSUE THREAD

**Issue by rsln-s** [tensorflow/serving#92]
This is not really an issue, but I need help and I didn't know where to ask for it.

Is it possible at current state of tensorflow serving to write iOS client that for example talks to MNIST model (as in [mnist_inference.cc](https://github.com/tensorflow/serving/tree/master/tensorflow_serving/example/mnist_inference.cc) and [mnist_client](https://github.com/tensorflow/serving/tree/master/tensorflow_serving/example/mnist_client.py))?

In other words, do I need the Tensorflow dependency for it or gRPC/protobufs are sufficient (since protobufs are used as a format for data transfers and we are basically talking to gRPC service)?

**Comment by nfiedel** [comment]
Hi rsln-s@,
  Are you thinking of having the TensorFlow model hosted on some remote server, or locally on the device?

  If on a remote server, you will only need gRPC/protobufs to talk with the server.

  If local, you will need TensorFlow.

  Thanks,
  Noah

**Comment by rsln-s** [comment]
Thanks for the answer.
Yes, the model will be hosted in a docker container on a remote server.
